### PR TITLE
Add event survey field for events in content editor (469)

### PR DESCRIPTION
**Pull request by sxpydo**
https://github.com/isaaccomputerscience/isaac-cs-issues/issues/469

### CODE REVIEW

File: src/services/emptyDocuments.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ const emptyDocuments: Document[] = [
     eventStatus: "CLOSED",
     preResources: [],
     postResources: [],
+    eventSurvey: [],
```
Comment (hannah-whelan) [inline src/services/emptyDocuments.ts:99]: Why is eventSurvey an array? Will there be multiple surveys for each event?

Comment (sxpydo) [reply src/services/emptyDocuments.ts:99]: There wouldn't be multiple surveys, however. would give the option incase they wanted to send more than one survey eventually?

Comment (sxpydo) [reply src/services/emptyDocuments.ts:99]: If this isn't the case, what would replace it?

File: src/services/emptyDocuments.ts [hunk 2]
```diff
@@ -96,7 +96,6 @@ const emptyDocuments: Document[] = [
     eventStatus: "CLOSED",
     preResources: [],
     postResources: [],
-    eventSurvey: [],
```
Comment (hannah-whelan) [inline src/services/emptyDocuments.ts]: Maybe want to include this in the emptyDocument, even if its just as "Enter survey link here..." or just ""

File: src/services/emptyDocuments.ts [hunk 3]
```diff
@@ -96,7 +96,6 @@ const emptyDocuments: Document[] = [
     eventStatus: "CLOSED",
```
Comment (hannah-whelan) [inline src/services/emptyDocuments.ts:96]: Not part of this PR, however it might be nice to update the default date here `date: 1442430000000` to be one that is this year (something like 1722470400000), so it's better for creating new dates. It doesn't really make sense to have this default to 2015 now.

File: src/isaac-data-types.d.ts [hunk 4]
```diff
@@ -123,7 +123,7 @@ export interface IsaacEventPage extends Content {
   location?: Location;
   preResources?: ExternalReference[];
   postResources?: ExternalReference[];
-  eventSurvey?: ExternalReference[];
+  eventSurvey?: string;
```
Comment (hannah-whelan) [inline src/isaac-data-types.d.ts:126]: Looks like you have manually modified the types file here? You'll need to make sure the changes are in the API and then regenerate the file using the mvn extension

**Review by hannah-whelan — APPROVED** [review]
